### DISCUSSION TITLE
error "cannot find symbol import com.swmansion.rnscreens.RNScreensPackage;" in task compileDebugJavaWithJavac

### DISCUSSION THREAD

**Discussion by dn1t** [software-mansion/react-native-screens#1638] [Q&A] [answered]
Hello, I'm trying to use this package with react-navigation 6.x. I've been struggling because of this issue for weeks, but couldn't find out why this happens. So I am just using the old version, but I also want to know the fundamental solution.

Using react-native@0.70.6, react-native-screens@3.18.2.

```
error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Volumes/CuteSSD/1_NonAndroid/Development/Working/TokiViewer/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
/Volumes/CuteSSD/1_NonAndroid/Development/Working/TokiViewer/android/app/build/generated/rncli/src/main/java/com/facebook/react/PackageList.java:19: error: cannot find symbol
import com.swmansion.rnscreens.RNScreensPackage;
                              ^
  symbol:   class RNScreensPackage
  location: package com.swmansion.rnscreens
/Volumes/CuteSSD/1_NonAndroid/Development/Working/TokiViewer/android/app/build/generated/rncli/src/main/java/com/facebook/react/PackageList.java:69: error: cannot find symbol
      new RNScreensPackage(),
          ^
  symbol:   class RNScreensPackage
  location: class PackageList
Note: /Volumes/CuteSSD/1_NonAndroid/Development/Working/TokiViewer/android/app/src/debug/java/com/tokiviewer/ReactNativeFlipper.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
```

**Comment by kkafar** [comment, marked as answer]
Hi @thoratica,

This seems like a material for an issue rather than a discussion, however I'll try to answer it here.

`RNScreenPackage` class is used by ReactNative CLI for autolinking & code generation purposes. Seeing this stacktrace indicates that application was not properly linked with the library. 

My first recommendation would be the most obvious one: clean your `node_modules` and build directories. Try running:

```
rm -fr node_modules yarn.lock android/build
yarn install
```

in your application directory. In case you're using `npm` and not `yarn`, remember to replace the command with one that appropriate for `npm`: `npm install` I believe. The same goes for `yarn.lock` file.

Then try to build your application.

If the issue still appears, please consider filing an issue in the repository or provide some more information on your configuration (a [reproduction](https://stackoverflow.com/help/minimal-reproducible-example) would be the best).
